lib/create-branch.js

```diff
@@ -106,7 +106,31 @@ module.exports = async (
 
       const isNpm = lockfilePath.includes('package-lock.json')
       try {
-        const { ok, contents } = await getNewLockfile(commit.content, oldLockfileContent, isNpm)
+        const { tokens, tokens_audit } = await dbs() // eslint-disable-line
+
+        /*
+        This is the structure of the tokens 'model'
+        _id: `${installationId}
+          tokens: {
+            ${repoId}: {
+            npm: ${token},
+            github: ${token}
+          }
+        }
+        */
+        let repositoryTokens = null
+        try {
+          repositoryTokens = await tokens.get(installationId)
+
+          const datetime = new Date().toISOString().substr(0, 19).replace('T', '_')
```

Comment:
The token_audit id look like
`123:one-lockfile:2018-11-16_14:46:00:read`
I'm not sure if the datetime is nice. Julia wants everything to be dashes.

Reply:
we’re all in favour of the format '20181116144600'